proto/osmosis/superfluid/superfluid.proto

```diff
@@ -29,6 +29,9 @@ message SuperfluidAsset {
   // AssetType indicates whether the superfluid asset is a native token or an lp
   // share
   SuperfluidAssetType asset_type = 2;
+  // For non-osmo native assets, we need a pool_id osmo/asset to determine the
```

Comment:
Should we allow this to be multihop, in-case we want to use an ASSET->USDC->OSMO route for pricing?

Reply:
moving this into its own PR